src/tasks/components/TaskRunsRow.tsx

```diff
@@ -85,6 +90,16 @@ class TaskRunsRow extends PureComponent<Props, State> {
     this.setState({isImportOverlayVisible: !this.state.isImportOverlayVisible})
   }
 
+  private handleRetry = () => {
+    const {retryTask, taskID, run} = this.props
+    try {
+      retryTask(taskID, run.id)
+      window.location.reload()
```

Comment:
Re rendering these page seems a bit drastic. Is there a way to simply get the runs afresh by dispatching a thunk rather than using the reload here? I ask because a page reload will make for a clunkier UX